commonDependencyVersionConstraints/build.gradle

```diff
@@ -90,6 +90,9 @@ dependencies {
     api group: 'org.testcontainers', name: 'testcontainers', version: testcontainers
     api group: 'org.testcontainers', name: 'toxiproxy', version: testcontainers
 
+    // make sure that this is compatible with the testcontainers version
```

Comment:
Does this comment needs to be followed up on?  Otherwise lets delete, it works right?

Reply:
Thanks Peter!  Scariest 5 minutes of my life between when I read this and verified that it does indeed work!.
I had meant to do that much earlier, but then got caught up.